### PR TITLE
feat: Ship KEDA v2.11.1 Helm chart

### PR DESCRIPTION
**Pull request by tomkerkhove**
### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO - [learn more](https://github.com/kedacore/charts/blob/main/CONTRIBUTING.md#developer-certificate-of-origin-signing-your-work))

Ships https://github.com/kedacore/charts/issues/481
Relates to https://github.com/kedacore/keda/issues/4743

### CODE REVIEW

**Comment by JorTurFer** [comment]
We will have to release KEDA v2.11.1 soon because [there is an important bug related with pausing](https://github.com/kedacore/keda/pull/4734) ScaledObjects, does waiting for it make sense?

**Comment by tomkerkhove** [comment]
Already done =)

File: http-add-on/README.md.gotmpl [hunk 1]
```diff
@@ -120,16 +120,6 @@ their default values.
...
-
-| Parameter | Type | Default | Description |
-|-----------|------|---------|-------------|
-{{- range .Values }}
-  {{- if hasPrefix "webhooks" .Key }}
-| `{{ .Key }}` | {{ .Type }} | {{ if .Default }}{{ .Default }}{{ else }}{{ .AutoDefault }}{{ end }} | {{ if .Description }}{{ .Description }}{{ else }}{{ .AutoDescription }}{{ end }} |
-  {{- end }}
-{{- end }}
-
 Specify each parameter using the `--set key=value[,key=value]` argument to
```
Comment (mkilchhofer) [inline http-add-on/README.md.gotmpl:123]: Why do you remove this?

This is the README config/template for the other chart

Comment (JorTurFer) [reply http-add-on/README.md.gotmpl:123]: It's because that chart doesn't have any webhook. Only `keda` char has

Comment (tomkerkhove) [reply http-add-on/README.md.gotmpl:123]: Yes, because it should not be there :)

There are no webhooks there so every helm-doc run it adds empty things

**Review by JorTurFer — APPROVED** [review]

**Pull request merged**